frontend/src/components/pages/explore/ExploreClubList.tsx

```diff
@@ -0,0 +1,196 @@
+import * as React from "react";
+import { BookInfo } from "../club_display/BookInfo";
+import { BookInterface } from "../../../services/backend_service_interface/backend_service_interface";
+import Box from "@material-ui/core/Box";
+import Button from "@material-ui/core/Button";
+import { ClubDescription } from "../club_display/ClubDescription";
+import { ClubInterface } from "../../../services/backend_service_interface/backend_service_interface";
+import { ContentWarnings } from "../club_display/ContentWarnings";
+import { createStyles } from "@material-ui/core/styles";
+import Dialog from "@material-ui/core/Dialog";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogContentText from "@material-ui/core/DialogContentText";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import HighlightOffIcon from "@material-ui/icons/HighlightOff";
+import { Link } from "react-router-dom"
+import { makeStyles } from "@material-ui/core/styles";
+import PageviewIcon from "@material-ui/icons/Pageview";
+import { Theme } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    button : {
+      marginLeft: theme.spacing(1),
+      marginTop: theme.spacing(1),
+      height: '35px',
+    },
+    buttonContainer: {
+      marginTop: theme.spacing(1),
+    },
+    club : {
+      marginTop: theme.spacing(3),
+      width: '900px',
+    },
+    clubContent: {
+      display: 'flex',
+      flexBasis: '100%',
+      flexWrap: 'wrap',
+      marginBottom: theme.spacing(2),
+      marginLeft: theme.spacing(2),
+      marginRight: theme.spacing(2),
+    },
+    clubHeader: {
+      display: 'flex',
+      flexBasis: '100%',
+      justifyContent: 'space-between',
+      marginBottom: theme.spacing(2),
+      marginLeft: theme.spacing(2),
+      marginRight: theme.spacing(2),
+      marginTop: theme.spacing(1),
+    },
+    clubTitle: {
+      marginBottom: theme.spacing(0),
+      maxWidth: '500px',
+    },
+    link: {
+      color: theme.palette.text.primary,
+      textDecoration: 'none'
+    },
+    listedClubsContainer: {
+      alignItems: 'center',
+      display: 'flex',
+      flexDirection: 'column',
+      marginBottom: theme.spacing(3),
+    },
+  }),
+);
+
+interface ExploreClubListProps {
+  clubsToDisplay: ClubInterface[],
+  handleJoinClub(clubId: string): void,
+}
+
+/**
+ * Displays up to the number of clubs that the user has requested to
+ * the page.
+ */
+export function ExploreClubList(props: ExploreClubListProps) {
+  const classes = useStyles();
+  const clubsToDisplay = props.clubsToDisplay;
+
+  const [joinClubAlertOpen, setJoinAlertOpen] = React.useState<boolean>(/* closed */ false);
```

Comment:
```suggestion
  const [joinClubAlertOpen, setJoinAlertOpen] = React.useState<boolean>(/* closed= */ false);
```
Parameter comments should have a '=' in them.